backend/src/Designer/Controllers/OptionsController.cs

```diff
@@ -251,4 +255,27 @@ public async Task<ActionResult> DeleteOptionsList(string org, string repo, [From
 
         return Ok($"The options file {optionsListId}.json has been deleted.");
     }
+
+    [HttpPost]
+    [Route("{optionsListId}/import")]
+    public async Task<ActionResult> ImportOptionsListFromOrg(string org, string repo, [FromRoute] string optionsListId, CancellationToken cancellationToken = default)
+    {
+        cancellationToken.ThrowIfCancellationRequested();
+        string developer = AuthenticationHelper.GetDeveloperUserName(HttpContext);
+
+        bool codeListExists = await _orgCodeListService.CodeListExists(org, developer, optionsListId, cancellationToken);
+        if (!codeListExists)
+        {
+            return NotFound($"The code list file {optionsListId}.json does not exist.");
+        }
+
+        List<Option> newOptionsList = await _optionsService.ImportOptionsListFromOrg(org, repo, developer, optionsListId, cancellationToken);
+
+        if (newOptionsList is null)
+        {
+            return NotFound($"The options file {optionsListId}.json already exists.");
```

Comment:
Foreslår å bruke HTTP status 409 Conflict i stedet for 404 Not Found.
```suggestion
            return Conflict($"The options file {optionsListId}.json already exists.");
```